src/devices/Bmx280/BmxBase.cs

```diff
@@ -282,7 +282,7 @@ private Temperature CompensateTemperature(int adcTemperature)
             //Formula from the datasheet
             //The temperature is calculated using the compensation formula in the BMP280 datasheet
             double var1 = ((adcTemperature / 16384.0) - (_calibrationData.DigT1 / 1024.0)) * _calibrationData.DigT2;
-            double var2 = ((adcTemperature / 131072.0) - (_calibrationData.DigT1 / 8192.0)) * _calibrationData.DigT3;
+            double var2 = Math.Pow(((adcTemperature / 131072.0) - (_calibrationData.DigT1 / 8192.0)), 2) * _calibrationData.DigT3;
```

Comment:
Could we do x*x instead of Math.Pow? there is a bit of performance gain that we could get if we do it that way.

Reply:
Oh for sure, didn't even know that made a difference, changed it.

Reply:
Math.Pow just does some additional checks because it is a method that needs to handle a lot of edge cases for two dobules that are passed in. Given that what we want to do here is simply to elevate the variable to the power of 2, we don't need those extra checks and we can do that faster just inline 😉